README.md

```diff
@@ -59,77 +56,28 @@ To update dependencies
 poetry update
 ```
 
-
 ## Run locally
 
-Either run a local containerized server:
+Run a local containerized server:
 
 ```sh
 docker image build . -t calhoun-test:0
 docker kill t1
 docker run -e FLASK_DEBUG=1 --rm -itd --name t1 -p 8080:8080 calhoun-test:0
 ```
-
-### or 
```

Comment:
I couldn't get it working and the steps call for the use of pip which I'm not sure we're supporting here; I can move these docs to a later section with an explanatory note if preferred

Reply:
That would be best to move it instead of deleting it. This does not need poetry really since there is a virtual environment created via venv, and only flask is installed, so pip is fine in that case